types/react-draft-wysiwyg/index.d.ts

```diff
@@ -60,6 +60,11 @@ export interface EditorProps {
         editorState: EditorState,
         eventTimeStamp: number,
     ): Draft.DraftHandleValue;
+    handleBeforeInput?(
```

Comment:
This prop is supposed to come from the original library's usage of `draft-js`?
Looking at the library's source code, they appear to filter out some props: https://github.com/jpuri/react-draft-wysiwyg/blob/51fecc1cdb1804b6232b66523be2d1c6a47ea3a9/src/Editor/index.js#L58
So I don't see why `handleBeforeInput` should be added.